nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/kinesis/stream/ConsumeKinesisStream.java

```diff
@@ -333,6 +345,7 @@ public class ConsumeKinesisStream extends AbstractAwsAsyncProcessor<KinesisAsync
             NUM_RETRIES,
             RETRY_WAIT,
             REPORT_CLOUDWATCH_METRICS,
+            OUTPUT_STRATEGY,
```

Comment:
For organizational purposes, it would be better to move `Output Strategy` immediately after the Record Reader and Record Writer properties.